flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -25,37 +25,52 @@ class SentryFlutterOptions extends SentryOptions {
   /// feature, as this is required to mark Sessions as Crashed.
   bool enableNativeCrashHandling = true;
 
-  int _autoSessionTrackingIntervalMillis = 30000;
-
   /// The session tracking interval in millis. This is the interval to end a
   /// session if the App goes to the background.
   /// See: [enableAutoSessionTracking]
+  @Deprecated('Use autoSessionTrackingInterval instead')
   int get autoSessionTrackingIntervalMillis =>
-      _autoSessionTrackingIntervalMillis;
+      autoSessionTrackingInterval.inMilliseconds;
 
+  @Deprecated('Use autoSessionTrackingInterval instead')
   set autoSessionTrackingIntervalMillis(int value) {
-    _autoSessionTrackingIntervalMillis =
-        value >= 0 ? value : _autoSessionTrackingIntervalMillis;
+    autoSessionTrackingInterval = value >= 0
+        ? Duration(milliseconds: value)
+        : autoSessionTrackingInterval;
   }
 
+  /// The session tracking interval. This is the interval to end a
+  /// session if the App goes to the background.
+  /// Default is 30 seconds/30000 milliseconds.
+  /// See: [enableAutoSessionTracking]
+  /// Always uses the given duration as a positiv timespan.
+  Duration autoSessionTrackingInterval = Duration(milliseconds: 30000);
```

Comment:
The default in all mobile SDKs is 5 seconds. Was it agreed to changed to 30?

Reply:
It was like this before, I only changed the data type. If it should be 5 seconds, we should change it now because we're doing a major update anyway

Reply:
Actually it's 30 seconds.
At least on iOS and Android (Java), see 
https://github.com/getsentry/sentry-cocoa/blob/2bfbbcdc11f84efd54498120e882d42e8bedd30e/Sources/Sentry/SentryOptions.m#L42

and 

https://github.com/getsentry/sentry-java/blob/a5f30b43b1dad2634ce020809f3b52e0d564a22a/sentry/src/main/java/io/sentry/SentryOptions.java#L205

Reply:
its been always 30 seconds, never agreed to be 5

Reply:
I'm getting old, yes it was always 30. I used to set to 5 to test while developing.